presto-orc/src/main/java/com/facebook/presto/orc/OrcWriterOptions.java

```diff
@@ -64,7 +68,8 @@ private OrcWriterOptions(
             int rowGroupMaxRowCount,
             DataSize dictionaryMaxMemory,
             DataSize maxStringStatisticsLimit,
-            DataSize maxCompressionBufferSize)
+            DataSize maxCompressionBufferSize,
+            OptionalInt zstdCompressionLevel)
```

Comment:
Maybe remove "zstd" prefix from the zstdCompressionLevel. Other compression algos should also support some kind of the compression level.

Reply:
I am mixed on this one. OrcReaderOption has zstdJniDecompressionEnabled. Other compressors are implemented in airlift and only ZstdJniCompression is implemented in Presto. Since the new option only works for Zstd, I would like to keep it consistent with OrcReaderOption.

Reply:
Actually, +1 on making this `compressionLevel`. `zstdJniDecompressionEnabled` is actually a pretty bad backdoor breaking the abstraction. On the other hand, it's also ugly to have ZSTD_JNI compression kind to distinguish from ZSTD. I think it's time to remove aircompressor's ZSTD implementation so that we can remove this flag.

Reply:
changed the name to compressionLevel and introduced the level for both Zstd and Zlib. Other compressors will not honor the level.